samcli/lib/providers/provider.py

```diff
@@ -494,8 +494,28 @@ def cors_to_headers(cors: Optional["Cors"]) -> Dict[str, Union[int, str]]:
         """
         if not cors:
             return {}
+
+        # Resource processing start here.
+        # The following code is based on the following spec:
+        # https://www.w3.org/TR/2020/SPSD-cors-20200602/#resource-processing-model
+
+        if not request_origin:
+            return {}
+
+        # cors.allow_origin can be either a single origin or comma separated list of origins
+        allowed_origins = cors.allow_origin.split(",") if cors.allow_origin else list()
```

Comment:
Is it possible to have both of these strings (`"origin1,origin2"` and `"origin1, origin2"` with a space) be passed in as valid origins?

Reply:
I do not think so, as this is coming from yaml parsers (providers?) in this format. I'll add a trim to be safe 👍